src/parquet/column/test-specialization.h

```diff
@@ -115,7 +115,8 @@ void PrimitiveTypedTest<TestType>::SyncValuesOut() {}
 
 template <>
 void PrimitiveTypedTest<BooleanType>::SyncValuesOut() {
-  std::copy(bool_buffer_out_.begin(), bool_buffer_out_.end(), values_out_.begin());
+  std::transform(bool_buffer_out_.begin(), bool_buffer_out_.end(), values_out_.begin(),
```

Comment:
We could also use a for loop here with a cast. Since this is test code I don't think it matters too much